### ISSUE TITLE
Fieldset: Wrong ErrorMessage role

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystem#410]
### Description of the bug

During #397 i noticed that `Fieldset` was only using an `ErrorMessage` with role [alertdialog](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alertdialog_role) for `a11y` of invalidation.

Using `role="alertdialog"` here seems abit wrong as its meant to be used for a error dialog that pops up and the user must must confirm or do some other action.

There is also missing aria-description (and other `a11y` atttributes?) for screen-readers `FieldSet`. A quick test using `CheckboxGroup` and VoiceOver MacOs in Edge (chromium), I was not read the "error message" displayed on screen.

### Steps To Reproduce

1. Navigate to `CheckboxGroup - med feilmelding` story
2. Activate Voiceover on MacOs
3. Expected error message to be read out

**Comment by mimarz** [comment]
We will just change the role for now and fix the other stuff in #519 as its a bigger job.
